Directory.Packages.props

```diff
@@ -8,8 +8,8 @@
         <MicrosoftExtensionPackageVersion>8.0.0</MicrosoftExtensionPackageVersion>
         <MicrosoftTestPackageVersion>17.8.0</MicrosoftTestPackageVersion>
         <MSBuildPackageVersion>17.3.2</MSBuildPackageVersion>
-        <NuGetPackageVersion>6.11.0-rc.110</NuGetPackageVersion>
-        <RoslynPackageVersion>4.12.0-1.24358.3</RoslynPackageVersion>
+        <NuGetPackageVersion>6.13.0-rc.95</NuGetPackageVersion>
+        <RoslynPackageVersion>4.13.0-3.24617.3</RoslynPackageVersion>
```

Comment:
This will need to be updated once https://github.com/dotnet/roslyn/pull/75312 merges